src/orquestra/sdk/_base/_in_process_runtime.py

```diff
@@ -86,7 +87,26 @@ def __init__(self):
     def _gen_next_run_id(self, wf_def: ir.WorkflowDef):
         return f"{wf_def.name}-{len(self._output_store) + 1}"
 
-    def create_workflow_run(self, workflow_def: ir.WorkflowDef) -> WfRunId:
+    def create_workflow_run(
+        self,
+        workflow_def: ir.WorkflowDef,
+        workspace_id: t.Optional[str] = None,
+        project_id: t.Optional[str] = None,
```

Comment:
It would be nice to have a type alias like `WorkspaceID` and `ProjectID`, especially given the fact we might have human friendly strings soon too.